### PR TITLE
Deprecating queue in favor of queues

### PR DESCRIPTION
**Pull request by mprazz**
`queue` has been deprecated in favor of `queues`, so our deployment files need updating 😊

### CODE REVIEW

File: charts/rasa-x/templates/rasa-config-files-configmap.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ data:
       username: "{{ .Values.rabbitmq.rabbitmq.username }}"
       password: ${RABBITMQ_PASSWORD}
       port: {{ default 5672 .Values.rabbitmq.service.port }}
-      queue: ${RABBITMQ_QUEUE}
+      queues: ${RABBITMQ_QUEUE}
```
Comment (tczekajlo) [inline charts/rasa-x/templates/rasa-config-files-configmap.yaml:36]: @mprazz we should change it only if used version of Rasa OOS is >= version that is supporting `queues`.

(I've put `1.9.0` as an example :D )
```
{{- if semverCompare ">= 1.9.0" .Values.rasa.tag -}}
      queues: ${RABBITMQ_QUEUE}
{{- else -}}
      queue: ${RABBITMQ_QUEUE}
{{- end -}}
```

**Review by tczekajlo — CHANGES_REQUESTED** [review]
One small thing 😄

File: charts/rasa-x/templates/rasa-config-files-configmap.yaml (hunk 1 above)
Comment (mprazz) [reply charts/rasa-x/templates/rasa-config-files-configmap.yaml]: Deprecation was introduced on `1.9.0` actually 😂 
Also added the list format for `queues`

**Review by tczekajlo — APPROVED** [review]
🚀

File: charts/rasa-x/templates/rasa-config-files-configmap.yaml (hunk 1 above)
Comment (tczekajlo) [reply charts/rasa-x/templates/rasa-config-files-configmap.yaml]: Cool, I fixed the template formating, the rest looks good 🚀

**Pull request merged**